AK/Slugify.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023, Gurkirat Singh <tbhaxor@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <AK/Slugify.h>
+#include <AK/StringView.h>
+
+namespace AK {
+ErrorOr<String> slugify(String const& _raw, char const glue)
+{
+    StringBuilder sb;
+    bool just_processed_space = false;
+
+    auto _raw_trimmed = TRY(_raw.trim_ascii_whitespace());
```

Comment:
why the underscores?

Reply:
> why the underscores?

You mean for variable names?

Reply:
> 2. _raw isn't a nice variable name. raw is fine, just don't start names with an underscore.

Reply:
Oh yes, idk how it was missed. I have fixed it in b0cc4ab17d2fc122953fcf7c4a0efb8b43cf1859